activity/views.py

```diff
@@ -0,0 +1,5 @@
+from django.shortcuts import render
+
+
+def index(request):
```

Comment:
this duplicates information in `community/urls.py` .  why do we need it here?  (what are the benefits of this duplication?)

If we have it here, can we somehow avoid having it also in `community/urls.py` ?  Maybe a feature request in django-distill?